src/OpenTok/Session.php

```diff
@@ -148,4 +158,12 @@ public function generateToken($options = array())
     {
         return $this->opentok->generateToken($this->sessionId, $options);
     }
+
+    /**
+     * @return mixed
```

Comment:
Should this be `@return bool`? And can we please add a description:

```suggestion
    /**
     * Whether <a href="https://tokbox.com/developer/guides/end-to-end-encryption">end-to-end encryption</a>
     * is set for the session.
     *
     * @return mixed
```